config/routes.rb

```diff
@@ -51,6 +51,8 @@
 
   get "/generate_204", to: "application#generate_204", status: 204
 
+  get "*not_found", to: "application#not_found_404", status: 404
```

Comment:
was macht "not_found" in `get "*not_found", to:`

Reply:
nichts